client/ayon_core/hosts/maya/plugins/create/create_animation_pointcache.py

```diff
@@ -0,0 +1,135 @@
+from maya import cmds
+
+from ayon_core.hosts.maya.api import lib, plugin
+
+from ayon_core.lib import (
+    BoolDef,
+    NumberDef,
+)
+from ayon_core.pipeline import CreatedInstance
+
+
+def _get_animation_attr_defs(cls):
+    """Get Animation generic definitions."""
+    defs = lib.collect_animation_defs()
+    defs.extend(
+        [
+            BoolDef("farm", label="Submit to Farm"),
+            NumberDef("priority", label="Farm job Priority", default=50),
+            BoolDef("refresh", label="Refresh viewport during export"),
+            BoolDef(
+                "includeParentHierarchy",
+                label="Include Parent Hierarchy",
+                tooltip=(
+                    "Whether to include parent hierarchy of nodes in the "
+                    "publish instance."
+                )
+            ),
+            BoolDef(
+                "includeUserDefinedAttributes",
+                label="Include User Defined Attributes"
```

Comment:
Would a tooltip be nice? Maybe this describes it?
```suggestion
                label="Include User Defined Attributes",
                tooltip=(
                    "Whether to include all custom maya attributes found "
                    "on nodes as attributes in the Alembic data."
                )
```